Colours_And_Fonts_style_v1.css

```diff
@@ -240,10 +346,11 @@ img {
 
 .footer {
 	grid-column-end: span 2;
-	background-color: #B3E8FF;
+	background-color: #FBCA01;

```

Comment:
I changed the colour of the footer from light blue to yellow as it looked better and it matches the Tawa College colours.